cpg-analysis/src/main/kotlin/de/fraunhofer/aisec/cpg/passes/IdentifierPass.kt

```diff
@@ -30,6 +30,7 @@ import de.fraunhofer.aisec.cpg.graph.Node
 import de.fraunhofer.aisec.cpg.helpers.SubgraphWalker
 
 /** This [Pass] adds a unique ID to each [Node] in the CPG. */
+@Deprecated("Use a reference to a Node instead")
 class IdentifierPass : Pass() {
```

Comment:
I think we can remove it - or do we have any more dependencies to this pass? I think the ordering was the only use-case for it.

Reply:
We weren't sure whether applications using the CPG as a library still use it (i.e., the Clouditor). But if that's not the case, I'll just remove it.